oci.go

```diff
@@ -68,29 +61,14 @@ func getContainerInfo(containerID string) (vc.ContainerStatus, string, error) {
 		return vc.ContainerStatus{}, "", err
 	}
 
-	matchFound := false
 	for _, podStatus := range podStatusList {
 		for _, containerStatus := range podStatus.ContainersStatus {
 			if containerStatus.ID == containerID {
 				return containerStatus, podStatus.ID, nil
 			}
-
-			if strings.HasPrefix(containerStatus.ID, containerID) {
-				if matchFound {
-					return vc.ContainerStatus{}, "", errPrefixContIDNotUnique
-				}
-
-				matchFound = true
-				cStatus = containerStatus
-				podID = podStatus.ID
-			}
 		}
 	}
 
-	if matchFound {
-		return cStatus, podID, nil
-	}
-
 	return vc.ContainerStatus{}, "", nil
```

Comment:
We could add a comment saying that not finding the container ID is a valid case since this function is used for both cases (when checking if we can create a new container and when checking for a specific container).